roboto/bot.py

```diff
@@ -1519,6 +1521,85 @@ async def get_my_commands(self) -> List[BotCommand]:
             await make_request(self.session, HTTPMethod.POST, '/getMyCommands'),
         )
 
+    async def edit_message_text(
+        self,
+        chat_id: Union[ChatID, str],
+        message_id: MessageID,
+        text: str,
+        parse_mode: Optional[ParseMode] = None,
+        disable_web_page_preview: Optional[bool] = None,
+        reply_markup: Optional[InlineKeyboardMarkup] = None,
+    ) -> Message:
+        """editMessageText API method (for normal messages).
```

Comment:
```suggestion
        """editMessageText API method (for non-inline messages).
```

Reply:
When I first looked at it, it was not clear to me what you meant by a normal message